Userland/Utilities/cal.cpp

```diff
@@ -148,16 +163,16 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     if (year_mode) {
         outln("{: ^{}}", TRY(String::formatted("Year {}", year)), year_width);
 
-        for (int i = 1; i < 12; ++i) {
+        for (int mon = 1; mon < 12; ++mon) {
```

Comment:
no need to abbreviate

Reply:
`month` is already taken though and I didn't want to unnecessarily shadow

Reply:
month_index seems fine

Reply:
Done